www/components/shelf/shelf.js

```diff
@@ -128,7 +128,7 @@ class Shelf extends LitElement {
 
       return html`
         <li class="list-wrap">
-          <a href="${item.link}" @click="${this.handleClick}"><h2 id="${id}">${item.label} <span>${chevron}</span></h2></a>
+        <a id="${id}" href="${item.link}" @click="${this.handleClick}"><h2>${item.label}</h2><span>${chevron}</span></a>
```

Comment:
minor indentation change.

also, I think a block element (`<h2>`) technically isn't allowed inside an inline element (`<a>`).  (I know not introduced by this PR but just commenting on it.)

Reply:
Well we might as well fix that while we're at it. A span would be valid.